discordjs/discordjs-commonjs/src/modules/Private/commands/Eval.js

```diff
@@ -6,8 +6,6 @@ const {
     CommandPermissions,
     CommandOptions,
     CommandResponse,
-    AxonEnums,
```

Comment:
why removing those?
I'd actually be enclined to import the other that are not imported here (all the Utils element for easy access in eval)

Reply:
This applies for all other eval class

Reply:
AxonEnums isn't actually detected by intellisense for some reason. I can include it if you want

Reply:
yeah I'd prefer if you include it. (tbh make this eval the same as your ts one as I mentionned in another comment)